client/src/planwise/client/scenarios/handlers.cljs

```diff
@@ -236,6 +236,21 @@
  (fn [db [_]]
    (update db :list asdf/invalidate!)))
 
+(rf/reg-event-db
+ :scenarios/change-sort-column
```

Comment:
Wouldn't be simpler to have 1 event and letting the "change sort order on the current column" be another change sort column with column + order data?